module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPassword.md

```diff
@@ -9,42 +18,60 @@ schema: 2.0.0
 # New-EntraBetaApplicationPassword
 
 ## Synopsis
+
 Adds a strong password to an application.
 
 ## Syntax
 
-```
-New-EntraBetaApplicationPassword -ObjectId <String> -PasswordCredential <PasswordCredential>
+```powershell
+New-EntraBetaApplicationPassword 
+ -ObjectId <String> 
+ -PasswordCredential <PasswordCredential>
  [<CommonParameters>]
 ```
 
 ## Description
+
 Adds a strong password to an application.
 
 ## Examples
 
 ### Example 1: Add a password to an application
+
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All','Directory.ReadWrite.All'
+$PasswordCredential= New-Object Microsoft.Open.MSGraph.Model.PasswordCredential
+$PasswordCredential.StartDateTime = Get-Date -Year 2024 -Month 12 -Day 28
+$PasswordCredential.EndDateTime = Get-Date -Year 2025 -Month 2 -Day 28
+$PasswordCredential.CustomKeyIdentifier = [System.Text.Encoding]::UTF8.GetBytes('a')
+$PasswordCredential.Hint = 'b'
+$PasswordCredential.DisplayName = 'mypassword'
```

Comment:
Confirm code logic is already implemented. The main branch does not have this param supported (displayName).

Reply:
Updated document according to the new PR
https://github.com/microsoftgraph/entra-powershell/pull/969